themes/default/content/docs/pulumi-cloud/deployments/api.md

```diff
@@ -1005,6 +1005,106 @@ Response
 }
 ```
 
+### Pause Deployments
+
+Pauses all new deployments for a stack or organization. Already running deployments are allowed to complete and are not paused. New deployments are queued, and will run when the stack or organization's deployments are resumed.
```

Comment:
Can you add clarity in that first sentence around pausing deployments in the queue, as that makes it more clear about running jobs that need to complete per our discussion. thx!

Reply:
Done!